timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -116,9 +116,9 @@ def get_default_hps(key, prediction_length):
                 "fail_if_misconfigured": True,
             },
             "ARIMA": {
-                "maxiter": 50,
-                "order": ag.Categorical((1, 1, 1), (2, 0, 1)),
-                "seasonal_order": ag.Categorical((1, 0, 0), (1, 0, 1), (1, 1, 1)),
+                "maxiter": ag.Categorical(50),
```

Comment:
it looks like we're not leaving any hyperparameter space to explore. can we at least have (1, 1, 0), (1, 0, 0) in `order`?

Reply:
I plan to soon add a reasonable search space for HPO with many more configurations, but that requires taking care of some other issues first. 

My idea was to keep the config exactly as-is to ensure that presets are not changing at all between 0.5.1, 0.5.2 and 0.5.3 (if it gets released soon), and then again for 0.5.4.